### ISSUE TITLE
Get all followers

### ISSUE THREAD

**Issue by thrize** [Steem-FOSSbot/what-tags#4]
Currently the app only gets the first 1000 followers. This should be paged per, say, 100 followers, and go on to get all followers.

**Comment by thrize** [comment]
Blocked by https://github.com/steemit/steem-js/issues/90

**Comment by thrize** [comment]
Done, merge into master
